package.json

```diff
@@ -1,29 +1,35 @@
 {
-  "name":             "taskcluster-aws-provisioner",
-  "version":          "0.1.0",
-  "private":          true,
-  "main":             "bin/server.js",
+  "name": "taskcluster-aws-provisioner",
+  "version": "0.1.0",
+  "private": true,
+  "main": "bin/server.js",
```

Comment:
I... don't think so.  My understanding is that this is just the file that node ought to look for when requiring this whole package when installed as a dependency.